src/pkg/packager/dev.go

```diff
@@ -105,3 +107,46 @@ func (p *Packager) DevDeploy(ctx context.Context) error {
 	// cd back
 	return os.Chdir(cwd)
 }
+
+// Lint ensures a package is valid & follows suggested conventions
+func (p *Packager) Lint(ctx context.Context) (err error) {
+	config.CommonOptions.Confirm = true
+	cwd, err := os.Getwd()
+	if err != nil {
+		return err
+	}
+
+	if err := os.Chdir(p.cfg.CreateOpts.BaseDir); err != nil {
+		return fmt.Errorf("unable to access directory %q: %w", p.cfg.CreateOpts.BaseDir, err)
+	}
+
+	pc := creator.NewPackageCreator(p.cfg.CreateOpts, cwd)
+
+	if err := helpers.CreatePathAndCopy(layout.ZarfYAML, p.layout.ZarfYAML); err != nil {
+		return err
+	}
+
+	p.cfg.Pkg, p.warnings, err = pc.LoadPackageDefinition(ctx, p.layout)
+	if err != nil {
+		return err
+	}
+
+	lintFindings, err := lint.Validate(ctx, p.cfg.CreateOpts)
+	if err != nil {
+		return fmt.Errorf("linting failed: %w", err)
+	}
+	p.warnings = append(p.warnings, lintFindings...)
+
+	if len(p.warnings) == 0 {
```

Comment:
🤔  Why print this - I think it could be left out

Reply:
It just feels like we should give feedback to the user so they know the lint actually happened. Screenshot of without vs with. 
![image](https://github.com/defenseunicorns/zarf/assets/37223396/63b7ee35-462a-44bc-b49e-94b7ad4fb1c4)